documentation/decisions/adr/2023-12-20-contact-us-email.md

```diff
@@ -0,0 +1,84 @@
+# Method and technology for "Contact Us" CTA
+
+- **Status:** {Active}
+- **Last Modified:** 2023-12-20
+- **Related Issue:** [#682](https://github.com/HHS/simpler-grants-gov/issues/682)
+- **Deciders:** Lucas Brown, Andy Cochran, Sumi Thaiveetil, {other collaborators}
+- **Tags:** <!-- OPTIONAL -->
+
+## Context and Problem Statement
+
+How might we give users an alternative to public comments (GitHub, Google Groups, listserv, etc.) so that they might submit questions or feedback more privately? Is email the appropriate method? And how might we manage this feedback channel?
+
+
+## Decision Drivers
+
+- The "Contact Us" method should not be (or feel) like a black box
+- Incoming messages should be easy/simple to triage, respond to, and escalate to the appropriate channels
+
+## Options Considered
+
+- Self-hosted .gov email address:
+  - simpler@grants.gov
+  - simplergrantsgov@hhs.gov
+- [Google collaborative inbox](https://support.google.com/a/users/answer/167430?hl=en)
+- Help Desk tool (such as):
+  - [Loop](https://www.intheloop.io/)
+  - [Front](https://front.com/)
+  - [Odoo CRM](https://www.odoo.com/app/crm)
+
+## Decision Outcome: simpler@grants.gov
+
+Chosen option: **simpler@grants.gov**, because it is simple and accessible for end users, can be easily managed by comms staff, and matches the Simpler.Grants.gov domain.
+
+### Positive Consequences
+
+- Simple email communication is a technology that most all users are fimiliar and comfortable with.
+- Since it does not require creation of accounts, or using 3rd-party ticketing dashboards, email is the most straightforward method by which users can sumbit their feedback.
```

Comment:
```suggestion
- Since it does not require creation of accounts, or using 3rd-party ticketing dashboards, email is the most straightforward method by which users can submit their feedback.
```